twilio.cabal

```diff
@@ -79,37 +79,37 @@ library
                        Twilio.UsageTrigger,
                        Twilio.UsageTriggers
   hs-source-dirs:      src
-  build-depends:       aeson >=0.11,
-                       base ==4.*,
-                       bifunctors >= 5 && <5.3,
-                       bytestring ==0.10.*,
-                       containers ==0.5.*,
-                       errors >=2 && <2.3,
-                       exceptions ==0.*,
-                       free ==4.*,
-                       http-client ==0.4.*,
-                       http-client-tls ==0.2.*,
-                       http-types ==0.*,
-                       mtl ==2.*,
-                       network-uri >=2.6,
-                       old-locale ==1.0.*,
-                       scientific ==0.*,
-                       text ==1.*,
-                       time >=1.5 && <1.7,
-                       transformers >=0.4 && <0.6,
-                       unordered-containers ==0.2.*
+  build-depends:       aeson,
+                       base,
+                       bifunctors,
+                       bytestring,
+                       containers,
+                       errors,
+                       exceptions,
+                       free,
+                       http-client,
+                       http-client-tls,
+                       http-types,
+                       mtl,
+                       network-uri,
+                       old-locale,
+                       scientific,
+                       text,
+                       time,
+                       transformers,
+                       unordered-containers
 
 test-suite Tests
   hs-source-dirs:      test
   main-is:             Test.hs
   Type:                exitcode-stdio-1.0
-  build-depends:       aeson >=0.11,
-                       base ==4.*,
-                       bytestring ==0.10.*,
-                       Cabal >=1.16.0,
-                       http-client ==0.*,
-                       http-client-tls ==0.2.*,
-                       network-uri >=2.6,
-                       text ==1.*,
-                       transformers >=0.4 && <0.6,
+  build-depends:       aeson,
+                       base,
+                       bytestring,
+                       Cabal,
+                       http-client,
+                       http-client-tls,
+                       network-uri,
+                       text,
+                       transformers,
```

Comment:
I don't know much about Stack, but in order for this package to continue working with vanilla Cabal/Hackage, don't we need to keep _some_ known version bounds in the `.cabal` file?